api/APIv2.md

```diff
@@ -246,13 +248,13 @@ Request:
+`ZAttestReq` has `reqType` which MUST be filled by EVE to identify type of the request. If `reqType` is `ATTEST_REQ_CERT`, `certs` should be filled with certificates used by EVE. If `isIMutable` attribute is not set, Controller should reject any request that would overwrite an existing certificate previously published. If `reqType` is `ATTEST_REQ_NONCE`, no other field needs to be set in `ZAttestReq`. If `reqType` is `ATTEST_REQ_QUOTE`, then `quote` MUST be populated with the following information: `attestData` containing [TPMS_ATTEST structure defined in TCG](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-2-Structures-01.38.pdf), `signature` containing signature of `TPMS_ATTEST` by the attestation signing key, `pcr_sha256_values` containing values of PCR 0-15, `event_log` containing TPM Event Log (aka measurement logs) with each event entry of the format `TpmEventLogEntry`,  `eve_version` containing current EVE version, `fw_version` containing the current firmware version, `gps_info` containing GPS co-ordinates of device's geo-location.
 
 Response:
 
-The request mime type MUST be "application/x-proto-binary".
+The response mime type MUST be "application/x-proto-binary".
 The body MUST be a protobuf message of type AuthContainer where the AuthBody is a protobuf message of type [attest.ZAttestResp](./proto/attest/attest.proto).
-`ZAttestResp` has `respType` which MUST be filled by Controller. If `respType` is `ATTEST_RESP_CERT`, no other field needs to be set. If `respType` is `ATTEST_RESP_NONCE`, nonce field MUST be set. If `respType` is `ATTEST_RESP_QUOTE`, then `quoteResp` MUST be filled with the result of the attestation.
+`ZAttestResp` has `respType` which MUST be filled by Controller. If `respType` is `ATTEST_RESP_CERT`, no other field needs to be set. If `respType` is `ATTEST_RESP_NONCE`, nonce field MUST be set. If `respType` is `ATTEST_RESP_QUOTE`, then `quoteResp` MUST be filled with the result of quote verification, with `ATTEST_RESPONSE_SUCCESS` on successful verification of PCR quote, `ATTEST_RESPONSE_NONCE_MISMATCH` indicating that nonce in the `attestData` is not matching nonce expected from controller, and `ATTEST_RESPONSE_QUOTE_FAILED` when quote provided did not pass verification checks in controller. Additionally, when `quoteResp` has response code as `ATTEST_RESPONSE_SUCCESS`, the response must also fill `integrity_token` providing a token to present in the subsequent config requests, and `volume_storage_key` containing the decryption key for the volume storage vault for unlocking volumes on the device.
```

Comment:
Is it called integrity_token or integrityToken in the protobuf? You use both spellings.

Reply:
Earlier it was integrityToken, but due to Yetus complaint, I renamed it to integrity_token. Missed updating it here. Updated it now, please check.